src/components/GroupedList/GroupedList.Props.ts

```diff
@@ -167,6 +167,9 @@ export interface IGroupHeaderProps {
 
   /** Callback for when the group is selected. */
   onToggleSelectGroup?: (group: IGroup) => void;
+
+  /** Determines if the group selection check box is shown for collapsed groups. */
+  preventSelectCollapsedGroups?: boolean;
```

Comment:
honestly i feel like we are baking too many proprietary things into groupedlist. The groupedlist should be able to render any header/footer. Those things have their own proprietary settings and look/feel. Shouldn't this be a header concern?